components/public-api-server/pkg/server/integration_test.go

```diff
@@ -38,9 +38,6 @@ func TestPublicAPIServer_v1_WorkspaceService(t *testing.T) {
 
 	workspaceClient := v1.NewWorkspacesServiceClient(conn)
 
-	_, err = workspaceClient.ListWorkspaces(ctx, &v1.ListWorkspacesRequest{})
```

Comment:
Would you be able to add a specific integration test for ListWorkspaces now that it's implemented, please?

Reply:
Done - I switched over to the gomock-generated `GitpodAPIInterface` implementation in the process. This allows for more fine-grained tests (e.g. passing the page size as limit).